tests/testthat/test-indentation_linter.R

```diff
@@ -632,6 +632,17 @@ test_that("hanging_indent_stlye works", {
   expect_lint(code_hanging_same_line, NULL, tidy_linter)
   expect_lint(code_hanging_same_line, NULL, hanging_linter)
   expect_lint(code_hanging_same_line, "Indent", non_hanging_linter)
+
+  # regression test for #1898
+  expect_lint(
+    trim_some("
+      outer_fun(inner_fun(x,
+        one_indent = 42L
+      ))
+    "),
+    NULL,
+    tidy_linter
+  )
```

Comment:
This looks good to me, @AshesITR! Thanks for the quick fix.

Do you think we should include any of the following test cases?

``` r
library(lintr)

"outer_fun(inner_fun(
  x,
  one_indent = 42L
))" -> code

lint(code, linters = indentation_linter())

"outer_fun(inner_fun(x, # this is first arg
  one_indent = 42L # this is second arg
))" -> code

lint(code, linters = indentation_linter())

"outer_fun(
  inner_fun(
    x,
    one_indent = 42L
  )
)" -> code

lint(code, linters = indentation_linter())
```

<sup>Created on 2023-04-04 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>